### PR TITLE
Modifying script for production build

### PR DESCRIPTION
**Pull request by iFlameing**
Relate to issue #10 

please make comment if I have to make more modification :)

### CODE REVIEW

**Comment by iFlameing** [comment]
@domoritz I added all the necessary config for hosting the theme website. You can see my repo website at this link https://iflameing.github.io/vega-themes/ :)

File: test/index.html [hunk 1]
```diff
@@ -2,10 +2,10 @@
 <html>
   <head>
     <title>Vega Theme Test</title>
-    <script src="../node_modules/vega-lib/build/vega.js"></script>
```
Comment (domoritz) [inline test/index.html]: Why test and not dist or something?

Comment (iFlameing) [reply test/index.html]: I should change that. should I change it to 

`Vega Theme Collection`.

what do you think?

**Comment by iFlameing** [comment]
@domoritz I didn't want to disturb you but if you open some issue I can try to solve them :)

File: test/index.html (hunk 1 above)
Comment (domoritz) [reply test/index.html]: Nm, this is good.

**Comment by domoritz** [comment]
Thank you.

**Comment by iFlameing** [comment]
@domoritz you can merge this now :)

File: test/index.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 <html>
   <head>
-    <title>Vega Theme Test</title>
-    <script src="../node_modules/vega-lib/build/vega.js"></script>
-    <script src="../node_modules/vega-lite/build/vega-lite.js"></script>
-    <script src="../node_modules/vega-embed/build/vega-embed.js"></script>
-    <script src="../build/vega-themes.js"></script>
+    <title>Vega Theme Collections</title>
+    <script src="https://cdn.jsdelivr.net/npm/vega@4"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega-lite@2"></script>
```
Comment (domoritz) [inline test/index.html]: Unfortunately, this is not an option for us as we often want to use a local build of these libraries. By using jsdelivr, we won't be able to that.

Comment (iFlameing) [reply test/index.html]: then we have to push the node_modules in the reop?

Comment (domoritz) [reply test/index.html]: No, we do not. We could have a separate index.html for the website for example. Or we could build a bundle (that's a lot of overhead, though).

Comment (iFlameing) [reply test/index.html]: what should I do now? And how I can push seperate index.html to gh-page branch.

Comment (domoritz) [reply test/index.html]: Oh, I was imagining a separate folder `dist` where you have an `index.html` file that uses jsdelivr instead of local files.

**Comment by domoritz** [comment]
This needs some work before we can merge it. Ping me when it's ready.

**Comment by iFlameing** [comment]
@domoritz I couldn't revert the above changes because it will break the deployed app next time. should I make a new directory dist and copy the files which are currently in the test folder and revert the above changes for a local build.

**Comment by domoritz** [comment]
You can just add to it. I don't mind squashing your code.

**Comment by iFlameing** [comment]
@domoritz ok

**Comment by iFlameing** [comment]
@domoritz I made all the changes as you said, I think now this is ready for merge.

**Pull request merged**